.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,10 +17,7 @@ I'm suggesting these edits to an existing topic or collection:
 - [ ] Image (and my file is `*.png`, square, dimensions 288x288)
 - [ ] Content (and my changes are in `index.md`)
 
-**Please explain why these changes are necessary:**
-
-<!-- Type in the reason below -->
-
+> Please delete this line and explain why you think these changes should be made.
```

Comment:
I guess it's the same meaning, maybe a little more imperative, but what do you think of:

> Please replace this line with an explanation of why you think these changes should be made.

Reply:
👍 dig it